earth_enterprise/src/fusion/dbroot/SConscript

```diff
@@ -67,13 +67,13 @@ gelayerjsgen = env.executable('gelayerjsgen',
 env.install('fusion_lib', [geprotorootgen])
 env.install('fusion_bin', [gedbrootgen, gerasterdbrootgen, gelayerjsgen])
 
-env.test('dbroot_generator_tests',
-         ['dbroot_generator_tests.cc'],
-         LIBS = ['geprotorootgen',
-                 'gedbroot',
-                 'geiconutil',
-                 'geutil',
-                 'gecommon',
-                 'qt-mt',
-                 'gtest',
-                 ])
+#env.test('dbroot_generator_tests',
```

Comment:
This is a temporary workaround and only applies to CentOS 6. That's why it's on a different branch than master. When all is said and done, it would detect that it's building on CentOS 6 and modify features as needed. There is a follow-up ticket for fixing gtest compile on CentOS 6, #355.